### PR TITLE
Add `complete_review` endpoint and `reviewer_ids` param to job locales

### PR DESCRIPTION
**Pull request by vermaxik**
Improve documentation:
- added POST `/projects/{project_id}/jobs/{job_id}/locales/{id}/complete_review`
- added `reviewer_ids` param to `Create a job locale` and `Update a job locale`
- recompile all docs with previous commits

### CODE REVIEW

File: paths/job_locales/complete_review.yaml [hunk 1]
```diff
@@ -0,0 +1,59 @@
+---
+summary: Review a job locale
+description: Marked Job locale as reviewed.
```
Comment (theSoenke) [inline paths/job_locales/complete_review.yaml:3]: I'd change it to `Mark job locale as reviewed`

File: schemas/job_locale.yaml [hunk 2]
```diff
@@ -28,9 +28,10 @@ job_locale:
       name: My Job 1
       state: completed
     users:
-    - id: abcd1234cdef1234abcd1234cdef1234
+      id: abcd1234cdef1234abcd1234cdef1234
```
Comment (theSoenke) [inline schemas/job_locale.yaml:31]: I think the dash is required as it's a list right?

Comment (vermaxik) [reply schemas/job_locale.yaml:31]: good point 👍  thanks, returned back

File: paths/job_locales/complete_review.yaml (hunk 1 above)
Comment (vermaxik) [reply paths/job_locales/complete_review.yaml:3]: 👍  changed, thx

**Pull request merged**